README.md

```diff
@@ -209,17 +209,26 @@ Currently, this supports following event type.
 
 Here is an example of trigger payload:
 ```
-[
-  {
-    "msg": "updated README to add the description for the function-A",
-    "author": "user.localhost2000@gmail.com",
-    "repository": "XAAS/deploy-test",
-    "branch": "master",
-    "time": "'2017-06-24 08:17:26'"
-  }
-]
+{
```

Comment:
It looks like the trigger format hasn't changed and it's not backward compatible, right?

Reply:
Actually, looking at the code changes below it looks like it didn't really change, just a new key was added and readme was out of date.